scripts/scil_compute_dti_metrics.py

```diff
@@ -341,26 +339,52 @@ def main():
         R_img = nib.Nifti1Image(R.astype(np.float32), affine)
         nib.save(R_img, args.residual)
 
-        R_k = np.zeros(data.shape[-1])  # mean residual per DWI
+        # Each volume's residual statistics
+        if args.mask is None:
+            logger.info("Outlier detection will not be performed, since no "
+                        "mask was provided.")
+        stats = [dict.fromkeys(['label', 'mean', 'iqr', 'cilo', 'cihi', 'whishi',
+                                'whislo', 'fliers', 'q1', 'med', 'q3'], [])
+                 for i in range(data.shape[-1])]  # stats with format for boxplots
```

Comment:
I think something is wrong here ?

Reply:
Can you explain? It's producing a list of dict for i in ...

Reply:
Ok my bad it's ok. I didn't see that it was for stats.